packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -95,7 +100,16 @@ module.exports = function (
         const form = request.payload as { email: string; oldAuthPW: string };
         const { oldAuthPW, email } = form;
 
-        await customs.check(request, email, 'passwordChange');
+        const sessionToken = request.auth.credentials;
```

Comment:
I'd be interested in recording a metric here to monitor when this endpoint is called without a sessionToken.